docs/security-policy-for-authors.md

```diff
@@ -0,0 +1,348 @@
+
+Also note on which conditions you may forward the security issue to
+other people.  For example,
+
+> The project maintainer(s) may forward this issue to the security
+> contacts for other projects where we believe it is relevant.  This
+> may include embedded libraries or prerequisite modules or system
+> libraries, or downstream software that uses this software.
+>
+> They may also forward this issue to CPANSec if they have not
+> already been copied on this.
+
+## What Software this Policy Applies to
+
+This section discusses what software this applies to, and what are
+considered security issues in the software, and what are not
+considered security issues.  For example,
+
+> Any security vulnerabilities in Foo-Bar are covered by this policy.
+>
+> Security vulnerabilities are considered anything that allows users
+> to execute unauthorised code, access unahtorised resources, or to
+> have an adverse impact on accessibility or performance of a system.
+>
+> Security vulnerabilities in upstream software (embedded libraries,
+> prerequisite modules or system libraries, or in Perl), are not
+> convered by this policy.
+>
+> Security vulnerabilities in downstream software (any software that
+> uses Foo-Bar, or plugins to it that are not included with the
+> Foo-Bar distribution) are not covered by this policy.
+>
+> However, vulnerabilities in upstream software that can affect
+> Foo-Bar, or where Foo-Bar can be used to exploit vulnerabilities in
+> other software (upstream or downstream), are considered
+> vulnerabilities of Foo-Bar, and are covered by this policy.
+
+### Which versions of this software are supported?
+
+There should be a section or subsection that identifies which versions
+of the software will be supported for security fixes.  For example,
+
+> The maintainer(s) will only commit to releasing security fixes for
+> the latest version of Foo-Bar.
+>
+> Note that the Foo-Bar project only supports major versions of Perl
+> released in the past ten (10) years, even though Foo-Bar will run on
+> older versions of Perl.  If a security fix requires us to increase
+> the minimum version of Perl that is supported, then we may do that.
+
+or,
+
+> For security vulnerabilities, the maintainer(s) will only commit to
+> supporting for the last two major versions of Foo-Bar.  As of this
+> release, that is v1.4.x and v1.6x.
+
+## Installation and usage issues
+
+It is also worth noting how it works with prerequisites, for example,
+
+> This software uses the zlib library for accessing gzip-encoded
+> streams and files.  The minimum requirement is zlib v1.2.  However,
+> there may be security vulnerabilties and you should use v1.2.13 or
+> newer.
+
+or
+
+> The distribution metadata specifies minimum versions of
+> prerequisites that are required for Foo-Bar to work.  However, some
+> of these prerequisites may have security vulnerabilities, and you
+> should ensure that you are using up-to-date versions of these
```

Comment:
Replace `up-to-date` with `non vulnerable`?

Reply:
Perhaps "versions with no known relevant vulnerabilities".